crates/sui-replay-2/src/lib.rs

```diff
@@ -558,6 +599,18 @@ where
             warn!("Failed to write data store summary: {:?}", e);
         }
     }
+
+    if digests.len() > 1 {
+        println!(
+            "Replay run: tx_count={} success={} failure={} - exec_ms={}, total_ms={}",
```

Comment:
I think this is probably fine, but my only concern is if we always display timings it makes it impossible for these outputs to be used in any kind of snapshot testing situation as your snapshots will always be different.

Reply:
Possibly have a flag or some other kind of setting that would not show timings?